### ISSUE TITLE
Continuous update for real-time application 

### ISSUE THREAD

**Issue by CSharpDummy** [fastlib/fCWT#61]
Is there a way to update the calculation when a new time series data point becomes available?
Or is the only way to calculate based on a rolling window of fixed size?

**Comment by fastlib** [comment]
For now, the only way is to calculate a new window that overlaps with the previous calculate time-frequency matrix. For 99% of real-world cases, this is sufficient. Think about this. If your lowest frequency is 1Hz, you only need to calculate a rolling window of 1s. If your frequency is way lower, say 0.01Hz, in theory, you need a window of 100s. However, you can ask yourself whether you want to recalculate such a frequencyband every second... In the end, it's a very slow moving wave which means it's 'quite' stationary for short timespans. 

In short, yes, the only way is using a moving window, but this is not necessarily bad due to the theory behind frequencies. 

Good luck!
